modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/DDMFormRendererHelper.java

```diff
@@ -103,6 +107,22 @@ protected DDMForm getDDMForm() {
 		return configurationModelToDDMFormConverter.getDDMForm();
 	}
 
+	protected DDMFormLayout getDDMFormLayout() {
+		Class<?> formClass =
+			ConfigurationDDMFormDeclarationUtil.getConfigurationDDMFormClass(
+				_configurationModel);
+
+		if (formClass != null) {
+			try {
+				return DDMFormLayoutFactory.create(formClass);
+			}
+			catch (IllegalArgumentException iae) {
+			}
+		}
+
+		return DDMUtil.getDefaultDDMFormLayout(getDDMForm());
```

Comment:
We should pass in the ddmForm that we already have from line 59 as a parameter instead of calling this again.